modules/extensions/iam.tf

```diff
@@ -17,11 +17,25 @@ locals {
   create_operator_instance_principal_dynamic_group = (var.use_cluster_encryption == true && var.create_policies == true && var.create_bastion_host == true && var.enable_operator_instance_principal == true)
 }
 
+resource "random_id" "dynamic_group_suffix" {
```

Comment:
Awesome. Can you please also implement this in the operator module?

Reply:
https://github.com/oracle-terraform-modules/terraform-oci-operator/pull/69